.buildkite/pipeline.yml

```diff
@@ -1,3 +1,7 @@
+env:
+  JULIA_PKG_SERVER: "" # it often struggles with our large artifacts
+  # SECRET_CODECOV_TOKEN: "..."
```

Comment:
FYI, The secret variables can be huge, once encrypted. That's why I had put it at the bottom. (see e.g. CUDA.jl)

Reply:
Ah fair point.

Reply:
Done.